src/components/20-molecules/image-upload/demo.js

```diff
@@ -0,0 +1,63 @@
+/* global document */
+import { storiesOf } from '@storybook/html';
+// import { html, render } from 'lit-html';
+import Readme from './README.md';
+import './index';
+
+storiesOf('Molecules/Image Upload/Demos', module)
+  .addParameters({
+    readme: {
+      sidebar: Readme,
+    },
+  })
+  .add('Feature - image upload in a form', () => {
```

Comment:
You could write the demo with lit-html, only for next time to keep it in mind.

Reply:
done